load-test-framework/src/main/java/com/google/pubsub/clients/gcloud/CPSPublisherTask.java

```diff
@@ -70,7 +70,7 @@ public void run() {
       pubSub.publish(topic, messages);
       stopwatch.stop();
       numberOfMessages.addAndGet(batchSize);
-      metricsHandler.recordLatency(stopwatch.elapsed(TimeUnit.MILLISECONDS));
```

Comment:
No, before I was multiplying by the batch size later, but it's definitely better if every Task does their exporting the same.